miniapp/io.cpp

```diff
@@ -179,6 +146,11 @@ cl_options read_options(int argc, char** argv, bool allow_write) {
         TCLAP::ValueArg<double> dt_arg(
             "d", "dt", "set simulation time step to <time> ms",
             false, defopts.dt, "time", cmd);
+        TCLAP::ValueArg<double> bin_dt_arg(
+            "", "bin-dt", "set event binning interval to <time> ms",
+            false, defopts.bin_dt, "time", cmd);
+        TCLAP::SwitchArg bin_regular_arg(
+            "","bin-regular","use 'regular' binning policy instead of 'following'", cmd, false);
```

Comment:
Totally want to rip out our option handling infrastructure and replace. _I have a plan_.

In the meantime, the hypens-in-long-options style is pretty common (it is certainly GNU style). Google test's underscore use is not standard practice. I think we should stick with it. We could modify the JSON keys to match, when we get around to this clean up.